docs/Getting Started/Adapting to the Head Unit Language/index.md

```diff
@@ -1,4 +1,5 @@
 # Adapting to the Head Unit Language
+@![iOS]
```

Comment:
Remove the iOS platform tag as this header works for both iOS and Java

Reply:
`Setting the Default Language` and the `What if My App Does Not Support the Head Unit Language?` sections are generic and should be visible on all platforms. (might have to add some inline platform tags)